backends/sycl-ref/ceed-sycl-ref-qfunction-load.sycl.cpp

```diff
@@ -0,0 +1,175 @@
+// Copyright (c) 2017-2022, Lawrence Livermore National Security, LLC and other CEED contributors.
+// All Rights Reserved. See the top-level LICENSE and NOTICE files for details.
+//
+// SPDX-License-Identifier: BSD-2-Clause
+//
+// This file is part of CEED:  http://github.com/ceed
+
+#include <ceed/backend.h>
+#include <ceed/ceed.h>
+#include <ceed/jit-tools.h>
+
+#include <iostream>
+#include <sstream>
+#include <string>
+#include <string_view>
+#include <sycl/sycl.hpp>
+#include <vector>
+
+#include "../sycl/ceed-sycl-compile.hpp"
+#include "ceed-sycl-ref.hpp"
+
+#define SUB_GROUP_SIZE_QF 16
+
+//------------------------------------------------------------------------------
+// Build QFunction kernel
+//
+// TODO: Refactor
+//------------------------------------------------------------------------------
+extern "C" int CeedSyclBuildQFunction(CeedQFunction qf) {
```

Comment:
This name we should fix after merging - its consistent with the CUDA and HIP backends, which both violate our usual convention, which would be `CeedQFunctionBuild_Sycl` or similar